website/docs/adopting.md

```diff
@@ -58,6 +60,30 @@ For small projects, Sorbet can run on a single file with no additional
 information. But for projects that have multiple files and depend on other gems,
 Sorbet needs to know more information to work.
 
+> **The `srb init` command is being deprecated.**
```

Comment:
Do you mind if I draft a quick blog post which we could include in this PR, and could contain a longer discussion of what would change, why the change is being made, timelines, etc.?